{
  "id": "MAL-2024-4721",
  "summary": "Malicious code in 18923aa (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:32:02Z",
  "published": "2024-06-25T13:32:02Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:32:02Z",
        "import_time": "2024-06-28T02:48:10.969032537Z",
        "versions": [
          "4.96"
        ],
        "id": "RLMA-2024-03501",
        "source": "reversing-labs",
        "sha256": "f5c68d6490005ccd26a9146bb5549a90ce6785dcad650fe7cc3e2e6804077007"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "18923aa",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/18923aa"
      },
      "versions": [
        "4.96"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/18923aa/MAL-2024-4721.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}